### DISCUSSION TITLE
Database export is garbled

### DISCUSSION THREAD

**Discussion by kingparks** [1Panel-dev/1Panel#7450] [Q&A]
### 1Panel Version

 社区版： v1.10.21-lts

### Problem Description

数据库备份中文乱码，期望能够默认带上参数  --default-character-set=utf8mb4 或者可以设置

### Steps to Reproduce

侧菜单->数据库->mysql->备份列表

**Comment by wanghe-fit2cloud** [comment]
备份的是本地数据库（由应用商店创建的 MySQL）还是远程数据库？

[a comment by an automated account is not shown]

**Comment by wanghe-fit2cloud** [comment]
本地数据库在创建时默认采用 utf8mb4 字符集，在进行备份时同样使用这一字符集，如图：
<img width="1475" alt="image" src="https://github.com/user-attachments/assets/37ada3f1-95b5-49f2-a8e1-51a476b446fd">

[a comment by an automated account is not shown]

**Comment by kingparks** [comment]
> 备份的是本地数据库（由应用商店创建的 MySQL）还是远程数据库？

远程数据库

[a comment by an automated account is not shown]

**Comment by wanghe-fit2cloud** [comment]
数据库备份远程数据库时，也会获取该数据库字符集，并在使用 mysqldump 命令备份时传入该字符集。
> 可以先检查一下该远程数据库的字符集是什么。

[a comment by an automated account is not shown]
